edx_proctoring/views.py

```diff
@@ -1022,24 +1034,23 @@ def _get_all_paginated_responses(self, backend, course_id, status_filters):
         else:
             status_filters = status_filters.split(',')
 
-        for status_filter in status_filters:
-            if status_filter is not None:
-                onboarding_profile_kwargs['status'] = VerificientOnboardingProfileStatus\
-                    .get_profile_status_from_filter(status_filter)
-            onboarding_profile_kwargs['page'] = 1
-            while onboarding_profile_kwargs['page'] is not None:
-                try:
-                    request = backend.get_onboarding_profile_info(
-                        course_id, **onboarding_profile_kwargs
-                    )
-                    results += request.get('results')
-                    onboarding_profile_kwargs['page'] = request.get('next_page_number')
-                except BackendProviderOnboardingProfilesException as exc:
-                    http_error = exc
-                    # return on the earliest error
-                    return results, http_error
-
-        return results, http_error
+        onboarding_status_filters = [
+            VerificientOnboardingProfileStatus.get_profile_status_from_filter(status_filter)
+            for status_filter in status_filters
+            if status_filter is not None
+        ]
+        onboarding_status_filter_string = ",".join(onboarding_status_filters)
+        onboarding_profile_kwargs['status'] = onboarding_status_filter_string
+
+        try:
+            request = backend.get_onboarding_profile_info(
```

Comment:
Before the try statement, could you log what is the content of `onboarding_profile_kwargs`?